WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
+        let service = BlogService(managedObjectContext: context)
+        let blog = service.blogByBlogId(settings?.primarySiteID)
+        
+        return blog?.settings?.name ?? String()
```

Comment:
@koke we cannot really return an optional here, because this is consumed by `EditableTextRow`, and that class requires `String`.

Regarding the `String()` over `""`, it's a personal decision: i'd rather being explicit. Literal vs new object instantiation.